src/circular-progress.jsx

```diff
@@ -73,12 +73,16 @@ let CircularProgress = React.createClass({
     if (this.props.mode !== "indeterminate") return;
 
     wrapper.style.transform = null;
+    wrapper.style.webkitTransform = null;
```

Comment:
You can use `AutoPrefix.set(wrapper.style, 'transform', null)` here. For reference see:
https://github.com/callemall/material-ui/blob/master/src/menus/menu.jsx#L85